src/uu/stat/stat.md

```diff
@@ -0,0 +1,64 @@
+# stat
```

Comment:
maybe we should open good first bugs to do this for all programs?
cc @tertsdiepraam

Reply:
Yep, good idea! This PR is a good intermediate step though.

Reply:
I don't think it makes sense for _all_ programs. For example, the help text of `who` contains some platform-specific information.

Reply:
Oh I see what you mean now, it's not just platform specific, but a value from a constant. It's tricky because ultimately, I don't really want these texts to require compiling the code. So we can read them from the markdown files directly to generate manpages and online docs, etc.. Anyway, let's start by converting at least all the utils that don't have platform specific text.

Reply:
I didn't want to make 100+ good first issues, so I just a GitHub task list thingy: https://github.com/uutils/coreutils/issues/4368

Shall I make it a pinned issue?

Reply:
sure, thanks :)